templates/6_tma_batch_qc_metrics.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<p>Same here, this makes it seem like the user has to do something. Make it clear where this fits into pipeline, what the user should have done already. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>